Sources/SwiftInspectorVisitors/Tests/StructVisitorSpec.swift

```diff
@@ -1,413 +0,0 @@
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooStruct.BarFooEnum.BarBarFooStruct") {
-            let matching = self.sut.structs.filter {
-              $0.name == "BarBarFooStruct"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooStruct.BarFooEnum"
-            }
-
-            expect(matching.count) == 1
-          }
-        }
-      }
-
-      context("visiting a code block with multiple top-level declarations") {
-        context("with multiple top-level structs") {
-          it("asserts") {
-            let content = """
-            public struct FooStruct {}
-            public struct BarStruct {}
-            """
-
-            // The StructVisitor is only meant to be used over a single struct.
-            // Using a StructVisitor over a block that has multiple top-level
-            // structs is API misuse.
-            expect(try VisitorExecutor.walkVisitor(
-                    self.sut,
-                    overContent: content))
-              .to(throwAssertion())
-          }
-        }
-
-        context("with a top-level class after a top-level struct") {
-          it("asserts") {
-            let content = """
-            public struct FooStruct {}
-            public class FooClass {}
-            """
-
-            // The StructVisitor is only meant to be used over a single struct.
-            // Using a StructVisitor over a block that has a top-level class
-            // is API misuse.
-            expect(try VisitorExecutor.walkVisitor(
-                    self.sut,
-                    overContent: content))
-              .to(throwAssertion())
-          }
-        }
-
-        context("with a top-level enum after a top-level struct") {
-          it("asserts") {
-            let content = """
-            public struct FooStruct {}
-            public struct FooEnum {}
-            """
-
-            // The StructVisitor is only meant to be used over a single struct.
-            // Using a StructVisitor over a block that has a top-level enum
-            // is API misuse.
-            expect(try VisitorExecutor.walkVisitor(
-                    self.sut,
-                    overContent: content))
-              .to(throwAssertion())
-          }
-        }
-      }
-
-      context("visiting a code block with a top-level class declaration") {
-        it("asserts") {
-          let content = """
-            public class FooClass {}
-            """
-
-          // The StructVisitor is only meant to be used over a single struct.
-          // Using a StructVisitor over a block that has a top-level class
-          // is API misuse.
-          expect(try VisitorExecutor.walkVisitor(
-                  self.sut,
-                  overContent: content))
-            .to(throwAssertion())
-        }
-      }
-
-      context("visiting a code block with a top-level enum declaration") {
```

Comment:
this test was not moved since a `NestableDeclarationVisitor` can parse this declaration type